Envfile

```diff
@@ -27,6 +27,11 @@ variable :RACK_TIMEOUT_SERVICE_TIMEOUT, :Integer, default: 100_000
 # Production related config that can be ignored
 variable :DEPLOYMENT_SIGNATURE, :String, default: "Optional"
 
+# For Redis session storage
+variable :REDIS_URL, :String, default: "redis://localhost:6379/0"
```

Comment:
`/0` in this URL is telling it to use db number 0. Since I would suspect the REDIS_URL will likely be used everywhere to connect to Redis in the future I think it would be better to remove the db number here and make that a session variable of its own or put it in the code.

Reply:
Chatted with @rhymes about this, can we change REDIS_URL to be REDIS_DB_URL to make it clear it points to a db?